readme.txt

```diff
@@ -85,5 +85,8 @@ Both installation methods expose the `wp core safe-update` WP-CLI command, which
 
 == Changelog ==
 
+### 0.1.1 (??? ??, 2018) ###
+* Updated 'closing </body> tag' detection message to clarify it's a success.
```

Comment:
0.1.0 is still unreleased, so you don't need to include this changelog entry.

Reply:
Thanks, should I remove fully or just move under 0.1.0?

Reply:
Thanks @danielbachhuber